src/usdb_syncer/settings.py

```diff
@@ -754,3 +851,39 @@ def get_path_template() -> path_template.PathTemplate:
 
 def set_path_template(template: path_template.PathTemplate) -> None:
     set_setting(SettingKey.PATH_TEMPLATE, template)
+
+
+def get_app_path(app: SupportedApps) -> str:
```

Comment:
Since this has to be a valid file path, we should use `Path`. (And maybe `None` to unset it?)

Reply:
Can I store a Path object in the settings, or do I convert to string before saving it to settings?

Reply:
Not sure if I did everything correctly here...